src/software/ai/passing/sampling_pass_generator.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <random>
+
+#include "base_pass_generator.h"
+#include "proto/parameters.pb.h"
+#include "software/ai/passing/cost_function.h"
+#include "software/ai/passing/pass_with_rating.h"
+#include "software/world/world.h"
+
+// the random seed used to initialize the random number generator
+static const int SEED = 14;
```

Comment:
```suggestion
static constexpr int SEED = 14;
```
`constexpr` gets inserted at compile-time